src/protagonist/API/Features/Queues/Requests/CreateBatchOfImages.cs

```diff
@@ -85,6 +89,8 @@ public class CreateBatchOfImagesHandler : IRequestHandler<CreateBatchOfImages, M
         var batch = await batchRepository.CreateBatch(request.CustomerId, request.AssetsBeforeProcessing.Select(a => a.Asset).ToList(), cancellationToken);
 
         var assetNotificationList = new List<Asset>(request.AssetsBeforeProcessing.Count);
+        var snsNotificationList = new List<AssetModificationRecord>();

```

Comment:
is prefixing this with `sns` leaking the underlying implmementation? It'll currently be published to an sns topic but that could change without calling code needing to know.

Reply:
I wanted something that would make it obvious it was doing something different from engine notification - best way to do this might be to change `assetNotificationList` to `engineNotificationList`, then changing this value to `assetModifiedNotificationList`